src/lib/apolloClient.js

```diff
@@ -0,0 +1,12 @@
+import { ApolloClient, InMemoryCache, gql } from "@apollo/client";
+import { THEGRAPH_ID } from "../constants";
+const APIRUL = "https://api.thegraph.com/subgraphs/id/"+THEGRAPH_ID;
```

Comment:
What's this constant name? Couldn't parse it kek

Reply:
lol, updated it to THE_GRAPH_ID